src/main/java/com/intuit/graphql/orchestrator/stitching/XtextStitcher.java

```diff
@@ -106,6 +108,7 @@ public RuntimeGraph stitch(List<ServiceProvider> serviceProviders) {
     Map<String, XtextGraph> xtextGraphMap = serviceProviders.stream()
         .map(XtextGraphBuilder::build)
         .map(xtextGraph -> transform(xtextGraph, preMergeTransformers))
+        .peek(this::validateGraph)
```

Comment:
just curious why validateGraph is after transform?

Reply:
Needs to be run after the type extension transformer. 
For example, if someone's schema looks 
type Query{}
extend type Query { getA: ID }
this is consider empty when first getting the resources and looking at the query, mutation, and subscriptions